app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -147,6 +147,10 @@
     <icon drawable="@drawable/messages" package="com.samsung.android.messaging" name="Messages" />
     <icon drawable="@drawable/messenger" package="com.facebook.mlite" name="Messenger" />
     <icon drawable="@drawable/messenger" package="com.facebook.orca" name="Messenger" />
+    <icon drawable="@drawable/messenger" package="com.android.messaging" name="Messaging" />
```

Comment:
One last thing: I believe `@drawable/messenger` should be `@drawable/messages` below. `@drawable/messenger` is for Facebook Messenger.

```suggestion
    <icon drawable="@drawable/messages" package="com.android.messaging" name="Messaging" />
```